homeassistant/components/epson/config_flow.py

```diff
@@ -25,18 +25,32 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_import(self, import_config):
         """Import a config entry from configuration.yaml."""
-        return await self.async_step_user(import_config)
+        if import_config is not None:
```

Comment:
The imported config should never be None. We can remove this check.

Reply:
ok, removed